cli/src/commands/pull/pull.test.ts

```diff
@@ -47,7 +47,77 @@ const baseFetch = (url: string, request: any) => {
       ok: true,
       json: async () => ({ schema: { tables: [{ name: 'table1', columns: [{ name: 'a', type: 'string' }] }] } })
     };
+  } else if (url === `https://test-1234.us-east-1.xata.sh/db/db1:main/schema` && request.method === 'GET') {
+    return {
+      ok: true,
+      json: async () => ({
+        schema: {
+          name: 'bb_hmtsb6hnd552p1rencda7oo3eg_3hae5b',
+          tables: {
+            table1: {
+              oid: '747164',
+              name: 'table1',
+              comment: '',
+              columns: {
+                a: {
+                  name: 'a',
+                  type: 'string',
+                  default: null,
+                  nullable: true,
+                  unique: false,
+                  comment: ''
+                },
+                _createdat: {
```

Comment:
These internal fields will be renamed to `xata_` soon so we might want to already rename them to the new format?